python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -1323,7 +1457,34 @@ router bgp {{ router_bgp.as }}
 {%     if router_bgp.address_family_ipv6 is arista.avd.defined %}
    !
    address-family ipv6
+{%         if router_bgp.address_family_ipv6.bgp.additional_paths.install is arista.avd.defined(true) %}
+      bgp additional-paths install
+{%         elif router_bgp.address_family_ipv6.bgp.additional_paths.install_ecmp_primary is arista.avd.defined(true) %}
+      bgp additional-paths install ecmp-primary
+{%         endif %}
+{%         if router_bgp.address_family_ipv6.bgp.additional_paths.receive is arista.avd.defined(true) %}
+      bgp additional-paths receive
+{%         endif %}
+{%         if router_bgp.address_family_ipv6.bgp.additional_paths.send is arista.avd.defined %}
+{%             if router_bgp.address_family_ipv6.bgp.additional_paths.send == 'disabled' %}
+      no bgp additional-paths send
+{%             elif router_bgp.address_family_ipv6.bgp.additional_paths.send == 'disabled_any' %}
+      no bgp additional-paths send any
+{%             elif router_bgp.address_family_ipv6.bgp.additional_paths.send_ecmp_limit is arista.avd.defined and router_bgp.address_family_ipv6.bgp.additional_paths.send == 'ecmp' %}
+      bgp additional-paths send ecmp limit {{ router_bgp.address_family_ipv6.bgp.additional_paths.send_ecmp_limit }}
+{%             elif router_bgp.address_family_ipv6.bgp.additional_paths.send == 'limit' %}
+{%                 if router_bgp.address_family_ipv6.bgp.additional_paths.send_limit is arista.avd.defined %}
+      bgp additional-paths send limit {{ router_bgp.address_family_ipv6.bgp.additional_paths.send_limit }}
+{%                 endif %}
+      bgp additional-paths send limit {{ router_bgp.address_family_ipv6.bgp.additional_paths.send_limit }}
```

Comment:
why do we have this 2 times ?
this is generating repeated commands

Reply:
Done.